module-5.md

```diff
@@ -12,36 +12,102 @@ Planned:      1.25 hour
+  * Run the speed test again, record the result
+  
+* Discuss
+  * What unit is the data usage of your apps measured in?
+  * What does MB mean?
+  * What was your top app and how much MB did it use?
+  * Was that surprising?
+  * Would you be more/less surprised if it was cellular data? What if it was WiFi data? 
+    * Are different data sources worth more?
+    * Why? What is your cell data limit?
+  
+  * What was the unit that the WiFi speed was measured in?
+  * What does Mbps mean?
+  * How does this relate to the MB from apps?
+  * Why was the speed different between tests?
+  
+* Show Grafana functioning on a node, and describe what it is
+  * Access via web interface
+  * Show how to select nodes
+  
+* Break into groups
+  * Connect to the node
+  * Access the Grafana interface and look around
+  * Stream a music file that is located on a node's local webserver and observe the metrics
+  * Stream a video file that is located on a node's local webserver and observe the metrics
+  * Observe what metrics change if you do the same from a remote node (over the mesh)
+  * Can you measure both nodes at the same time?
+
+* Discuss results
+  * Did difficulty increase measuring two nodes at the same time?
+  * What would happen if you had a citywide mesh?
+  * Are some metrics more important than others?
+  * How do you represent a mesh network?
+    * Physical network topology (i.e. Toronto Mesh map)
+    * Virtual peering topology (i.e. fc00.org)
 
 ## 3: Discussion and reflection activity
 
-Q & A about sharing broken internet experiences.
+Q & A about sharing broken internet experiences, understanding failure points and why we measure
+  * How would you be affected if the internet went down at home?
+    - Do you have alternate sources to connect to the internet (i.e. cellphone/LTE)?
+    - Do you wait for a user to complain that the internet is down or can you be instantly notified by monitoring your network?
+  * Have you ever had internet issues that were not complete outages? 
+    - Packet losses, poor signal quality, how much load time can you tolerate
+    - What are the most popular services on your network? How many users are accessing it and what is the available bandwidth?
+    - Do you need to setup additional nodes to offload bandwidth if a current node is over taxed with traffic?
+    - Who are the heaviest users? Do you need to introduce bandwidth metering? How do you prioritize bandwidth and users?
+    - Does network traffic change throughout the day?
+  * Port Traffic - clients can communicate over ports and port monitoring may be required if there is excessive traffic
+  * Gamers - why is low ping important? What impacts your ping?
+  * Review what a ping is and packet loss rates and how to use ping as a troubleshooting tool
+  * Other impacts on network health include routing loops, excessive broadcasts, denial of service attacks, viruses
+  * Baseline Measurements
```

Comment:
@Pedro-on-a-bike How are these questions asked and answered? What does "your network" refer to, is it the home network of everyone in attendance, or they are building a hypothetical network? I am a little confused about the flow of this _Discussion and reflection activity_ section.

Reply:
Perhaps it makes sense to divide into groups by use case (e.g. gamers, netflixers, casual browsers that just want minimal internet at reasonable cost, video conference for work all the time, ...) and have them come up with key metrics / capacity that will suit their use case?

Reply:
I think we need to distinguish between what you currently have (internet at home), and what you could have (mesh) would make it flow easier.

Reply:
Yes that differentiation is not currently made, so the questions feel very confusing.